app/views/layouts/_escalating_call_to_action.html.erb

```diff
@@ -7,7 +7,7 @@
         <% message = 'Upgrade to Premium Mob for group sessions with a Mentor' %>
     <% end %>
 <% else %>
-    <% path = new_user_registration_path %>
+    <% path = '/premium' %>
```

Comment:
this is for when people are not logged in - we don't want to promote premium here - we want the `new_user_registration_path` - this shouldn't change